### PR TITLE
【Hackathon No.108】Add Gelu operator, ffi, kernel for cpu and gpu.

### PR DESCRIPTION
**Pull request by KuangjuX**
This PR implements the cuda kernel implementation, operator definition, and FFI interface with Python for the Gelu function. **However, due to the lack of support for version 20 of ONNX operator definition in the Python ONNX library, the usage of the Gelu operator in python onnx library is still not supported.**

This PR has been successfully tested cuda kernel on NVidia A100 with CUDA 11.8 and CUDNN 8.7.

Related issue: https://github.com/PaddlePaddle/Paddle/issues/57262

### CODE REVIEW

**Comment by YdrMaster** [comment]
`master` 更新了，需要 merge 或 rebase

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,5 @@ build_debug/
 *.onnx
 *.pb
 *.npy
+
+.devcontainer
```
Comment (whjthu) [inline .gitignore:48]: 这种非大家通用的文件是不是就不往 .gitignore 里放了？

File: src/kernels/cuda/unary.cu [hunk 2]
```diff
@@ -121,12 +130,21 @@ void sqrt_kernel(float *input, float *output, size_t num) {
     int gridsize = (num + block_work_size() - 1) / block_work_size();
     _sqrt_kernel<<<gridsize, blocksize>>>(input, output, num);
 }
+
+void gelu_kernel(float *input, float *output, size_t num) {
+
```
Comment (whjthu) [inline src/kernels/cuda/unary.cu:135]: 空行删除

**Review by whjthu — CHANGES_REQUESTED** [review]

**Review by YdrMaster — APPROVED** [review]
重构之后可以改善这种加一个算子要改十几个文件的情况

**Review by whjthu — APPROVED** [review]

**Pull request merged**